CapstoneProject/Assets/Scripts/Core/Audio/SoundManager.cs

```diff
@@ -34,15 +34,22 @@
 public class SoundManager : PersistentObject<SoundManager>
 {
 
+    public SoundList SoundList;
```

Comment:
you should avoid naming a variable the exact same with cases and all as the data type.